pkg/image/api/validation/validation.go

```diff
@@ -308,6 +308,9 @@ func ValidateImageStreamImport(isi *api.ImageStreamImport) field.ErrorList {
 			if len(spec.From.Name) == 0 {
 				errs = append(errs, field.Required(imagesPath.Index(i).Child("from", "name"), ""))
 			} else {
+				if spec.From.Name == "*" {
```

Comment:
I wasn't expecting this to be allowed in the API, only used client side. What will allowing this in the API do?